src/plugin.js

```diff
@@ -2,6 +2,20 @@ import videojs from 'video.js';
 import window from 'global/window';
 import document from 'global/document';
 
+/** THe logic below is used to check if flash is disabled or not installed
+ *  in IE browser and show the appropriate message to the user.
+ */
+let isSupported = window.navigator.mimeTypes['application/x-shockwave-flash'];
+
+isSupported = isSupported && isSupported.enabledPlugin;
+let flashDisabled;
+
+if (isSupported === undefined) {
```

Comment:
you can write this if statement better without just assigning boolean values directly.